docs/en/02_Developer_Guides/01_Templates/11_Partial_Template_Caching.md

```diff
@@ -0,0 +1,282 @@
+---
+title: Partial Template Caching
+summary: Cache a section of a template Reduce rendering time with cached templates and understand the limitations of the ViewableData object caching.
+icon: tags
+---
+
+## Partial template caching
+
+Partial template caching is a feature that allows caching of rendered portions of templates. Cached content
+is fetched from a [cache backend](../performance/caching), instead of being regenerated repeatedly.
+
+
+### Base syntax
+
+Here is an example of the basic usage. This is not a definitive example of the syntax, but it shows the most
+common use case.
+
+[note]
+See also [Complete Syntax definition](#complete-syntax-defintition) section
+[/note]
+
+```ss
+<% cached $CacheKey if $Condition %>
+  <!-- Content block -->
+<% end_cached %>
+```
+
+The key parts are `$CacheKey`, `$Condition` and `<!-- Content block -->`.  
+The following sections explain every one of them in more detail.
+
+
+#### $CacheKey
+
+The `$CacheKey` is a string identifying the block content within the cache storage.
```

Comment:
Need to mention this requires a `getCacheKey()` function on Page_Controller or subclass

Reply:
Well, no it does not really require anything like that.
This construction is logical and should be chosen on case by case basis. I'm not sure how we could make it more explicitly in the doc though.

Reply:
PageSubClass_Controller::getCacheKey() is pretty much standard on our internal projects.  Once the cache key gets beyond the generic ['pagetype', $LastEdited, .count()] type people tend to use controllers because it's way cleaner than doing things in the template.  Also I've seen plenty of example of the people doing the ['pagetype', $LastEdited, .count()] within a getCacheKey() function bececause it's just makes the template look cleaner

Reply:
Either way, `$CacheKey` in this document is not talking about any particular implementations and may be any valid template expression (as long as it follows the form explained below).